topo/node/cisco/cisco.go

```diff
@@ -405,6 +408,18 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 		if pb.Config.Image == "" {
 			pb.Config.Image = "xrd:latest"
 		}
+		if pb.HostConstraints == nil {
+			pb.HostConstraints = append(pb.HostConstraints,
+				&tpb.HostConstraint{Constraint: &tpb.HostConstraint_KernelConstraint{
+					KernelConstraint: &tpb.KernelParam{Name: "fs.inotify.max_user_instances",
+						ConstraintType: &tpb.KernelParam_BoundedInteger{BoundedInteger: &tpb.BoundedInteger{
+							MaxValue: 64000, MinValue: 0}}}}})
```

Comment:
I think this is a min cons, so leave max blank and it should default to infinity in the checking

Reply:
FIxed.